apps/fluent-tester/src/E2E/Badge/pages/BasicBadgePageObject.macos.ts

```diff
@@ -1,40 +1,40 @@
-import {
-  BADGE_TESTPAGE,
-  HOMEPAGE_BADGE_BUTTON,
-  BADGE_TEST_COMPONENT,
-  BADGE_SECONDARY_TEST_COMPONENT,
-} from '../../../TestComponents/Badge/consts';
-import { BasePage, By } from '../../common/BasePage.macos';
+// import {
```

Comment:
Are you keeping this around for later?

Reply:
Did Win32 first, just added a commit for mac